catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/data/AttributeGroupType.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.forms.data;
+
+import com.google.common.collect.ImmutableSet;
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.impl.MetacardTypeImpl;
+import ddf.catalog.data.impl.types.CoreAttributes;
+import ddf.catalog.data.impl.types.SecurityAttributes;
+import java.util.Set;
+
+/**
+ * A shareable list of attribute descriptor names. They can represent sets of interesting fields.
+ *
+ * <p><i>This code is experimental. While it is functional and tested, it may change or be removed
+ * in a future version of the library.</i>
+ */
+public class AttributeGroupType extends MetacardTypeImpl {
+  public static final String ATTRIBUTE_GROUP_TAG = "attribute-group";
+
+  public static final String ATTRIBUTE_GROUP_LIST = "groups.attribute.list";
```

Comment:
❗️ This does not appear to follow the taxonomy naming conventions.  I think it would be `ui.attribute-group`.  Multivalued of true lets you know it is a list without it having to be part of the attribute name.

https://codice.atlassian.net/wiki/spaces/CODICE/pages/77627393/Taxonomy+Guidelines

@codice/data